koku/masu/external/downloader/aws/aws_report_downloader.py

```diff
@@ -177,6 +176,10 @@ def _get_manifest(self, date_time):
             (Dict): A dict-like object serialized from JSON data.
 
         """
+        # Checking if initial ingest is for a HCS manifest DO NOT PROCESS
+        if "Parquet" == self.report.get("Compression"):
+            LOG.info("Skipping ingest since data is Parquet compressed and likely for HCS")
```

Comment:
So does this bypass Parquet support for now unless they are going the customer filtered path? I think that is ok, but it would be nice to support more at some point.

Reply:
Right this essentially means a customer can add a bucket that has Parquet reports in (used with athena) And we just skip ingest. On the assumption that they really want to be posting the minimal report instead. Are you saying we should just straight support parquet files too? That feels like a lot more work :grin:

Reply:
Not with this feature, but it would be nice to support parquet at some point.